service/controller/key/key.go

```diff
@@ -398,6 +402,15 @@ func ToNodeCount(v interface{}) (int, error) {
 	return nodeCount, nil
 }
 
+func ToOperatorVersion(v interface{}) (string, error) {
+	customObject, err := ToCustomObject(v)
```

Comment:
As a side note we should get the naming straight here https://github.com/giantswarm/fmt/blob/master/go/variable_names.md#custom-resource

Reply:
Fixed here https://github.com/giantswarm/azure-operator/pull/680